bin/spritezero

```diff
@@ -41,11 +42,14 @@ if (argv.help || argv._.length < 2) {
 }
 
 var ratio = 1;
+var unique = false;
 
 if (argv.retina) {
   ratio = 2;
 } else if (argv.ratio) {
   ratio = parseFloat(argv.ratio);
+} else if (argv.unique) {
```

Comment:
This shouldn't be an exclusive option to argv.ratio - else if will mean that if `argv.ratio` is defined, the argv.unique option will not have an effect, which is incorrect.